debian/changelog

```diff
@@ -1,3 +1,9 @@
+rdma-core (15.1-1) unstable; urgency=low
+
+  * New upstream stable release.
+
+ -- Nicolas Morey-Chaisemartion <nmoreychaisemartin@suse.com>  Mon, 27 Nov 2017 10:37:05 +0100
+
 rdma-core (15-1) unstable; urgency=low
 
```

Comment:
We are not actually maintaing any of the packaging change logs in rdma-core upstream, so you can just revise the last entry to have the right version number.

It is necessary in debian for the changelog version number to match the version number in the control file.